cmd/build/command.go

```diff
@@ -90,6 +96,17 @@ func Build(ctx context.Context) *cobra.Command {
 func (bc *Command) getBuilder(options *types.BuildOptions) (Builder, error) {
 	var builder Builder
 
+	if options.File != "" && validateDockerfile(options.File) == nil {
```

Comment:
Why this change?

Reply:
i think this is to check the args in the case the file is passed as an option, we only accept 1 context 

`okteto build api -f Dockerfile` - this is allowed
`okteto build api frontend -f Dockerfile` i think this is not 

maybe i'm wrong... but i recall something regarding args validation was done inside command and i dont find now where was this 🤷🏻‍♀️

Reply:
But is it related to the the issue we are fixing?

Reply:
but then, you are changing the behavior for V2, aren't you?

Reply:
no because the logic it is only executed when we face a `manifest v1`, for that reason this _if condition_, to only show that error if it is a `manifest v1`

Reply:
Why this condition means only v1?

Reply:
from what I understood if -f == Dockerfile then we face a manifest v1. But that condition doesn't mean that we are going to catch all manifest v1 with it so I'm going to check if it is needed throw and error using the builder type instead of the current condition. I wanted to throw an error before call `loadContext` as previously implementation but at the moment it is not possible without a refactor of the command so it will be checked before calling `loadContext`.

Reply:
yeah, my concern was about the implicit knowledge we need to have to see that `-f == Dockerfile then we face a manifest v1.` when, in reality, we instantiate the builder based on the manifest parsed, not in the parameters received by the command.

Reply:
comments have been added on everything that requires a deeper knowledge that a priori is not available by reading the command.